cpp/CMakeLists.txt

```diff
@@ -249,6 +248,31 @@ if (RMM_INCLUDE AND RMM_LIBRARY)
     set_target_properties(rmm PROPERTIES IMPORTED_LOCATION ${RMM_LIBRARY})
 endif (RMM_INCLUDE AND RMM_LIBRARY)
 
+###################################################################################################
+# - External Projects -----------------------------------------------------------------------------
+
+# https://cmake.org/cmake/help/v3.0/module/ExternalProject.html
+include(ExternalProject)
+
+set(CUGUNROCK_DIR ${CMAKE_CURRENT_BINARY_DIR}/cugunrock CACHE STRING
+  "Path to cugunrock repo")
+
+ExternalProject_Add(cugunrock
+  GIT_REPOSITORY    https://github.com/rapidsai/cugunrock.git
+  GIT_TAG           fea_full_bc      # provide a branch, a tag, or even a commit hash
```

Comment:
Question : Do we have to edit the GIT_TAG each time gunrock changes?

Reply:
We have the option to do that, or we can just specify `master` and pull in the latest cugunrock changes that get pushed to `master` every time we rebuild.  This is currently set to `fea_full_bc` until cugunrock `master` has the features of that branch (build a static library, accept an install target).